ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/DcatShaclValidator.java

```diff
@@ -28,7 +31,13 @@ public boolean supports(Class<?> clazz) {
 
 	@Override
 	public void validate(Object target, Errors errors) {
-		Model model = (Model) target;
-		validateShape(model);
+		if (isValidatorActive()) {
+			Model model = (Model) target;
+			validateShape(model);
+		}
+	}
+
+	private boolean isValidatorActive() {
+		return StringUtils.isNotBlank(dcatShapeUri);
```

Comment:
Does not blank also cover not null?

Reply:
Checks if a CharSequence is not empty (""), not null and not whitespace only.
Whitespace is defined by Character.isWhitespace(char).
  StringUtils.isNotBlank(null)      = false
  StringUtils.isNotBlank("")        = false
  StringUtils.isNotBlank(" ")       = false
  StringUtils.isNotBlank("bob")     = true
  StringUtils.isNotBlank("  bob  ") = true